File: python/7 kyu/178.py
# https://www.codewars.com/kata/59fb783bab11f89202001083/train/python

# Passed

def recycle_me(rubbish):
    recycling_boxes = [0] * 3
    for trash in rubbish:
        if trash > 0:
            recycling_boxes[0] += 1
        elif trash < 0:
            recycling_boxes[1] += 1
        else:
            recycling_boxes[2] += 1
    
    recycling_boxes = tuple(recycling_boxes)
    return recycling_boxes

output = recycle_me([5, -9, 0, 6, -84, -95, 15]) 
print(output)

File: nodejs/7 kyu/178.js
// https://www.codewars.com/kata/5937ae46377144bb2f000029/train/javascript

// Passed

function getRectangleString(width, height) {
  let rows = [];
  for (let rowCount = 1; rowCount <= height; rowCount++) {
    let row;
    if (rowCount === 1 || rowCount === height) {
      row = "*".repeat(width);
    } else {
      row = width === 1 ? "*" : "*" + " ".repeat(width - 2) + "*";
    }

    rows.push(row);
  }

  let rectangle = rows.join("\r\n") + "\r\n";
  return rectangle;
}

const output = getRectangleString(5, 7);
console.log(output);